templates/repo/issue/view_content/comments.tmpl

```diff
@@ -732,14 +732,22 @@
 				{{template "shared/user/avatarlink" dict "Context" $.Context "user" .Poster}}
 				<span class="text grey muted-links">
 					{{template "shared/user/authorlink" .Poster}}
-					{{if gt .OldProjectID 0}}
-						{{if gt .ProjectID 0}}
-							{{$.locale.Tr "repo.issues.change_project_at" (.OldProject.Title|Escape) (.Project.Title|Escape) $createdStr | Safe}}
-						{{else}}
-							{{$.locale.Tr "repo.issues.remove_project_at" (.OldProject.Title|Escape) $createdStr | Safe}}
-						{{end}}
+					{{$oldProjectDisplayHtml := "Unknown Project"}}
```

Comment:
Can these be translated?

Reply:
How about `projects.type-0.display_name`

Reply:
I didn't do so because it's just like the deleted user (Ghost User) / deleted team (#24127), at the moment I don't see a strong requirement to do so.

Maybe we can leave it until enough people asking for such translation? It is really a rare case, which doesn't affect most/all users.

Reply:
Ok we should translate them all then in one batch, if we decide to do so.